Instagram/View Controllers/ProfileViewController.m

```diff
@@ -46,6 +46,8 @@ - (void)loadUserData {
             [self.profileImageView setImage:profileImage];
         }
     }];
+    [self.profileImageView.layer setCornerRadius:50];
```

Comment:
Since these are setters, you can also use dot notation here

```
self.profileImageView.layer.cornerRadius = 50.f;
self.profileImageView.layer.masksToBounds = YES;
```